plugins/BEdita/API/tests/TestCase/Controller/ObjectsControllerTest.php

```diff
@@ -65,20 +65,22 @@ public function testIndex()
                     'attributes' => [
                         'status' => 'on',
                         'uname' => 'first-user',
-                        'locked' => true,
-                        'created' => '2016-05-13T07:09:23+00:00',
-                        'modified' => '2016-05-13T07:09:23+00:00',
-                        'published' => null,
                         'title' => 'Mr. First User',
                         'description' => null,
                         'body' => null,
                         'extra' => null,
                         'lang' => 'eng',
-                        'created_by' => 1,
-                        'modified_by' => 1,
                         'publish_start' => null,
                         'publish_end' => null,
                     ],
+                    'meta' => [
+                        'locked' => true,
```

Comment:
Both fields tell you that a resource is not modifiable. Even if your not an `administrator` is probably a good thing to know.
That said, don't know where to show them.
Having a different layout for different roles could be confusing too...